src/scales/scale.time.js

```diff
@@ -477,6 +485,14 @@ module.exports = Scale.extend({
 			console.warn('options.time.format is deprecated and replaced by options.time.parser.');
 		}
 
+		if (time.min) {
+			console.warn('options.time.min is deprecated. Please use options.ticks.min');
```

Comment:
We could save a few bits on these messages:

```javascript
function deprecated(value, previous, current) {
    if (value !== undefined) {
        console.warn(
            'time scale: "' + previous+ '" is deprecated. ' +
            'Please use "' + current + '" instead');
    }
}

// ...

update: function() {

    // ...

    // DEPRECATIONS: output a message only one time per update
    deprecated(time.format, 'time.format', 'time.parser');
    deprecated(time.min, 'time.min', 'ticks.min');
    deprecated(time.max, 'time.max', 'ticks.max');
}
```

Reply:
Done